diff --git a/dist/index.mjs b/dist/index.mjs
index 800006118105cd58707159d0a98dc5329a719dc3..9048678fce39d66d56b5bad63b7c7c94a5e3dc94 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -11166,7 +11166,8 @@ function toMiniflareRequest(request$1) {
 		headers: [["accept-encoding", "identity"], ...request$1.headers],
 		body: request$1.body,
 		duplex: "half",
-		signal: request$1.signal
+		signal: request$1.signal,
+		credentials: "omit"
 	});
 }
 const isRolldown = "rolldownVersion" in vite;
